### PR TITLE
Minor additions to README.md

### PR DESCRIPTION
**Pull request by moeabdol**
I changed some of the wording in README.md, also added some additional bullet points in the Note on Patches/Pull Requests secition. I hope this makes it clearer for every one.

PS: This is my first OSF pull request 😃

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ Open source is made by people just like you. This Friday, invest a few hours con
 
 This repository is the source code for https://ossfriday.com
 
-Open Source Friday is a movement to encourage companies, people and maintainers to contribute a few hours contributing to open source every Friday.
+Open Source Friday is a movement to encourage people, companies,and maintainers to contribute a few hours to open source software every Friday.
```
Comment (Zarthus) [inline README.md]: There should be a space between 'companies,' and 'and' here

Comment (moeabdol) [inline README.md]: I agree. Thanks for pointing that out 👍

**Comment by MikeMcQuaid** [comment]
Thanks for the PR! Would love to get https://github.com/ossfriday/ossfriday/pull/115 landed first and then rebase on top of that.

**Comment by MikeMcQuaid** [comment]
Thanks @moeabdol, you rock!

**Pull request merged**